codice/alliance
# CAL-45 add mpeg2 support to alliance-mpegts-stream

#### What does this PR do?
Supports the chunking of mpeg2 udp streams.

#### Who is reviewing it?
@jrnorth @beyelerb @kcwire @jaymcnallie 
#### How should this be tested?
Install the experimental version of alliance.
Configure alliance to receive a udp video stream.
Generate an MPEG2 video stream (i can you help you with that).
Search the catalog and verify separate chunks are being created.
#### Any background context you want to provide?
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Checklist:
